client/js/app.js

```diff
@@ -31,5 +31,5 @@ if (module.parent) {
     }
   }
 } else {
-  app.start('#app-root')
+  app.start('#app-root', {href: false})
```

Comment:
right now that doesn't happen yet, actually. It's so that the  '/' route (hitting the home logo) goes home